### DISCUSSION TITLE
How to do a GraphQL query with Mavo?

### DISCUSSION THREAD

**Discussion by GalinhaLX** [mavoweb/mavo#937] [Q&A] [answered]
Hi,

In this page https://mavo.io/get/, since v.0.2.4, it seems we can do GraphQL queries with mavo:

> mv-storage-query (and mv-source-query/mv-init-query) attribute now can be used to make a GraphQL query when using the Github backend

Is there an example somewhere to see how it works?

Thanks

**Comment by DmitrySharabin** [comment, marked as answer]
Hey @GalinhaLX,

Yes, we can perform GraphQL queries with Mavo. You may find helpful this Lea's pen: https://codepen.io/leaverou/pen/NWbKBjL.

Please note that Lea don't use the `mv-source-query` backend attribute in her pen. But if she did, it would look like that: `mv-source="https://api.github.com/graphql" mv-source-query='query{ repository(owner: "w3ctag", name:"design-reviews") {...} }'`.
